namespaces/http.yml

```diff
@@ -0,0 +1,59 @@
+version: 2016.03.10.0
+
+defaults:
+  type: string
+  doc_values: true
+  index: not_analyzed
+
+namespace:
+  name: http
```

Comment:
Are these apache logs?  Might be better to call this `apache` unless it is a standard for all http servers - nginx, etc.